package/post-install.sh

```diff
@@ -26,28 +26,49 @@ ddtrace.request_init_hook=${EXTENSION_AUTO_INSTRUMENTATION_FILE}
 EOF
 }
 
-function install_conf_d_file() {
+function install_conf_d_files() {
     INI_FILE_PATH="${EXTENSION_CFG_DIR}/$INI_FILE_NAME"
 
-    println "Creating ddtrace.ini"
-    println "\n"
+    if [[ ! -f $INI_FILE_PATH ]]; then
```

Comment:
This file is by convention autogenerated - and we need to be able to recreate it when e.g. file paths would update (or PHP version changes in between installations).

Reply:
Allowing custom configuration should be possible though, but I think it could be done in other place. E.g. via adding another file like `99-z-ddtrace.ini` in php installtion which . would be able to  override specific options